incubator/hnc/pkg/testutils/testutils.go

```diff
@@ -144,19 +145,38 @@ func CleanupNamespaces(nses ...string) {
 	// has cascading deletion so we can delete any of its subnamespace descendants, and
 	// make sure that it's not a subnamespace itself so we can delete it directly.
 	for _, ns := range nses {
-		TryRunQuietly("kubectl hns set", ns, "-a")
-		TryRunQuietly("kubectl annotate ns", ns, "hnc.x-k8s.io/subnamespaceOf-")
-		TryRunQuietly("kubectl delete ns", ns)
+		TryRun("kubectl hns set", ns, "-a")
+		TryRun("kubectl annotate ns", ns, "hnc.x-k8s.io/subnamespaceOf-")
+		TryRun("kubectl delete ns", ns)
```

Comment:
Why? This can produce a lot of spam.

Reply:
+1

Reply:
I think it only outputs when the test failed and won't appear if the test passes. I mentioned it in the commit/PR message as "Change some `TryRunQuietly()` function in testutils package to
`TryRun()` so that if there's a failed test, we can see the output."

WDYT?

Reply:
I guess? `TryRunQuietly` has only ever been used for lines that we _expect_ to fail even in successful runs, so they could cause red herrings. Did you run into a case where this wasn't true? If not, I think I'd prefer to leave these alone.

Reply:
Oh I was trying to see what could be the issue of a namespace not being deleted. The issue is gone and I changed it back to quietly.